chroma-common/chroma-common.spec

```diff
@@ -17,19 +17,36 @@ Vendor: Intel(R) Corporation
 %description
 Common library containing routines used by both agent and manager.
 
+%package test
+Summary: Common test utilities for Intel Manager for Lustre tests
+Group: Development
+Requires: %{name} = %{version}-%{release}
+%description test
+This package contains shared test utilities used in the test framework for Manager for Lustre.
+
 %prep
 %setup -n %{name}-%{version}
 
 %build
-%{__python2} setup.py build
+%{__python} setup.py build
 
 %install
 rm -rf %{buildroot}
-%{__python2} setup.py install --skip-build --install-lib=%{python2_sitelib} --root=%{buildroot}
+%{__python} setup.py install --skip-build --install-lib=%{python_sitelib} --root=%{buildroot}
 mkdir -p $RPM_BUILD_ROOT/usr/sbin/
 
+touch test.files
+for test_file in $(find -L $RPM_BUILD_ROOT%{python_sitelib}/chroma_common/test/ -type f -name '*.py*'); do
+  install_file=${test_file/$RPM_BUILD_ROOT\///}
+  echo "${install_file}" >> test.files
+done
+
```

Comment:
good point, corrected in corrected in https://github.com/intel-hpdd/intel-manager-for-lustre/pull/76